workbench-core/authorization/src/caslAuthorizationPlugin.test.ts

```diff
@@ -149,15 +150,9 @@ describe('CASL Authorization Plugin', () => {
       'Random array should throw error',
       [fc.uniqueArray(fc.anything()), fc.uniqueArray(fc.anything(), { minLength: 1 })],
       async (userPermissions, operations) => {
-        try {
-          await caslAuthorizationPlugin.isAuthorized(
-            userPermissions as Permission[],
-            operations as Operation[]
-          );
-          expect.hasAssertions();
-        } catch (err) {
-          // eslint-disable-next-line no-empty
-        }
+        await expect(
+          caslAuthorizationPlugin.isAuthorized(userPermissions as Permission[], operations as Operation[])
+        ).rejects.toThrow();
```

Comment:
Do we know the error type? So we can put something like `rejects.toThrow(SomeExactErrorClass)`?